src/_premake_init.lua

```diff
@@ -1096,6 +1096,13 @@
 		kind = "boolean",
 	}
 
+	api.register {
+		name = "nuget",
+		scope = "project",
+		kind = "list:string",
+		tokens = true,
+	}
+
```

Comment:
Small nitpick, but I'm trying to keep these API definitions alphabetized. I'd like to add some automated JavaDoc style documentation at some point, and this way the APIs will appear alphabetized in the generated docs.